tests/test_tokenization_fast.py

```diff
@@ -90,3 +91,21 @@ def test_training_new_tokenizer_with_special_tokens_change(self):
                     # is restored
                     shutil.rmtree(self.tmpdirname)
                     self.tmpdirname = tmpdirname_orig
+
+
+@require_tokenizers
+class ReduceMutableBorrowTests(unittest.TestCase):
+    def test_async_share_tokenizer(self):
+        tokenizer = PreTrainedTokenizerFast.from_pretrained("gpt2")
+        tokenizer.add_special_tokens({"pad_token": "[PAD]"})
```

Comment:
As I have the impression here that the test is not really dependent on the tokenizer tested, I suggest you use this dummy tokenizer which is much smaller than gpt2 and already has the token `[PAD]` filled in. :slightly_smiling_face: 
```suggestion
        tokenizer = PreTrainedTokenizerFast.from_pretrained("robot-test/dummy-tokenizer-wordlevel")
```

Reply:
Thanks !